platform/android/app/build.gradle

```diff
@@ -104,6 +107,14 @@ android {
         }
     }
 
+    buildFeatures {
+        // Enables Jetpack Compose for this module
+        compose true
```

Comment:
But are we going to use it all over or just here?

Reply:
JetPack compose will be used to develop any new UI features, and probably migrate the old screens. 
This is mostly done to bring Android on par with iOS's SwiftUI; so we have more or less similar concepts.